src/test/java/net/snowflake/ingest/streaming/internal/RowBufferTest.java

```diff
@@ -1730,78 +1719,4 @@ public void testOnErrorAbortRowsWithError() {
     Assert.assertEquals(1, snapshotAbortParquet.size());
     Assert.assertEquals(Arrays.asList("a"), snapshotAbortParquet.get(0));
   }
-
-  @Test
-  public void testParquetChunkMetadataCreationIsThreadSafe() throws InterruptedException {
```

Comment:
lets keep this test around if its testing for thread safety?

Reply:
added back and compare the column data types instead of file names